commit 336094f43d82d83f374116862a0f568fe9092224
Author: Bhushan Shah <bhush94@gmail.com>
Date:   Mon Mar 9 21:04:09 2015 +0530

    Fix tabbar not showing up when switching to favorites from search
    
    when we are in search view tabbar is not visible and immediatly
    switching to favorites causes no tabbar.
    
    BUG: 343524
    FIXED-IN: 5.2.2

diff --git a/applets/kickoff/package/contents/ui/FullRepresentation.qml b/applets/kickoff/package/contents/ui/FullRepresentation.qml
index d978987..0ba18e6 100644
--- a/applets/kickoff/package/contents/ui/FullRepresentation.qml
+++ b/applets/kickoff/package/contents/ui/FullRepresentation.qml
@@ -493,6 +493,10 @@ Item {
                 target: root
                 Keys.forwardTo: [tabBar.layout]
             }
+            PropertyChanges {
+                target: tabBar
+                visible: true
+            }
         },
         State {
             name: "Applications"
@@ -500,6 +504,10 @@ Item {
                 target: root
                 Keys.forwardTo: [root]
             }
+            PropertyChanges {
+                target: tabBar
+                visible: true
+            }
         },
         State {
             name: "Search"
